api/models/Item.cs

```diff
@@ -44,6 +44,7 @@ public int PackOrderQuantity
     // Foreign Key Relationship
     public Guid? ItemGroupId { get; set; }
     public ItemGroup? ItemGroup { get; set; }
+    public Guid? SupplierId { get; set; }
 
```

Comment:
Je moet nog de navigation property toevoegen voor Supplier
```suggestion
public Supplier? Supplier{ get; set; }
```